ui/changes/src/graph.js

```diff
@@ -21,80 +21,36 @@ function generateData(count, yrange) {
   return series;
 }
 
-function rerender(connections, teamComponentMapping) {
+async function generateData() {
+  let map = await getComponentRegressionMap();
+  let return_map = [];
+  Object.keys(map)
+    .slice(0, 10)
+    .forEach((element) => {
+      let values = {};
+      values.data = [];
+      values.name = element;
+      for (const value in map[element]) {
+        let obj = {};
+        obj.x = value;
+        obj.y = Math.trunc(map[element][value] * 100);
+        values.data.push(obj);
+      }
+      return_map.push(values);
+    });
+  return return_map;
+}
+
+async function rerender(connections, teamComponentMapping) {
```

Comment:
A chart in the style of https://github.com/mozilla/bugbug/issues/2090#issuecomment-767033208 would be nicer to read (the lightness of the colors would show the importance of a link, while with the current UI there are just two levels)

Reply:
If you do something like https://apexcharts.com/javascript-chart-demos/heatmap-charts/basic/, you shouldn't need to choose the increments beforehand, but the lightness of the colors will show the strength of the link (the lighter the color, the smaller the link).